currency/code.go

```diff
@@ -272,35 +276,27 @@ func (c Code) MarshalJSON() ([]byte, error) {
 
 // IsEmpty returns true if the code is empty
 func (c Code) IsEmpty() bool {
-	if c.Item == nil {
-		return true
-	}
-	return c.Item.Symbol == ""
+	return c.Item == nil || c.Item.Symbol == ""
 }
 
-// Match returns if the code supplied is the same as the corresponding code
-func (c Code) Match(check Code) bool {
+// Equal returns if the code supplied is the same as the corresponding code
+func (c Code) Equal(check Code) bool {
 	return c.Item == check.Item
 }
 
-// IsDefaultFiatCurrency checks if the currency passed in matches the default
-// fiat currency
-func (c Code) IsDefaultFiatCurrency() bool {
-	return storage.IsDefaultCurrency(c)
-}
-
-// IsDefaultCryptocurrency checks if the currency passed in matches the default
-// cryptocurrency
-func (c Code) IsDefaultCryptocurrency() bool {
-	return storage.IsDefaultCryptocurrency(c)
-}
-
 // IsFiatCurrency checks if the currency passed is an enabled fiat currency
 func (c Code) IsFiatCurrency() bool {
-	return storage.IsFiatCurrency(c)
+	return c.Item != nil && c.Item.Role == Fiat
 }
 
 // IsCryptocurrency checks if the currency passed is an enabled CRYPTO currency.
+// NOTE: All unset currencies will default to cryptocurrencies and stable coins
+// are cryptocurrencies as well.
```

Comment:
I'm curious about this note. Is this intentional or because `c.Item.Role&(Cryptocurrency|Stable) == c.Item.Role` not being accurate enough?

Reply:
Yeah, intentional for all currencies that have a role as unset to be a cryptocurrency. I thought this might be the best way for now, as we don't specify what roles they are at compilation. And then we can set smaller subsets of currencies when we need to. 

As for not being accurate enough, the cryptos can be UNSET|STABLE|CRYPTOCURRENCY so any of those roles ***should be*** accurately depicted.